### PR TITLE
Updating readme with new components.

### PR DESCRIPTION
**Pull request by lizcross**
Adding some of the new components to the readme (forgot to add them during the prs)

JIRA: None
Linked PRs: None

## Changes
- Update the readme with the new components (components were added in a previous pr)

## How to test-drive this PR
- None

## TODOS:
- [x] +1 from an engineer on the Astro team.)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -95,9 +96,11 @@ Astro defines a controller inside of a requirejs module. The module exposes a pr
     * `app/scaffold-controllers/welcomeController.js` manages the navigation and layout of the welcome screen
```
Comment (kschmidtdev) [inline README.md]: maybe fix the path for this and the bottom two - it's really relevant to this PR but would save another tiny one? :)

**Comment by kschmidtdev** [comment]
:+1:

**Comment by tony-dinh** [comment]
I actually made all these README changes in my branch :D

**Comment by tony-dinh** [comment]
:+1:

**Pull request merged**